Tests/SonarQube.TeamBuild.PreProcessor.Tests/AnalysisConfigGeneratorTests.cs

```diff
@@ -133,13 +118,11 @@ public void AnalysisConfGen_AnalysisConfigDoesNotContainSensitiveData()
             Assert.AreEqual("1.0", config.SonarProjectVersion, "Unexpected project version");
 
             AssertExpectedLocalSetting(SonarProperties.HostUrl, "http://host", config);
-            AssertExpectedLocalSetting("file.public.key", "file public value", config);
+            AssertExpectedServerSetting("server.key.1", "server value 1", config);
+
+            AssertFileDoesNotContainText(config.FileName, "file.public.key"); // file settings values should not be in the config
 
-            // Sensitive arguments should not be in the file
-            AssertSettingDoesNotExist(SonarProperties.SonarUserName, config);
-            AssertSettingDoesNotExist(SonarProperties.SonarPassword, config);
-            AssertSettingDoesNotExist(SonarProperties.DbUserName, config);
-            AssertSettingDoesNotExist(SonarProperties.DbPassword, config);
+            AssertFileDoesNotContainText(config.FileName, "secret"); // sensitive data should not be in config
```

Comment:
It's simpler and more comprehensive to check that whole file rather than checking specific properties.